CHANGES.md

```diff
@@ -82,6 +82,7 @@ Release Notes.
 * Introduce the entity of Process type.
 * Set the length of event#parameters to 2000.
 * Limit the length of Event#parameters.
+* Introduce booster-ui and implement template management protocol.
```

Comment:
Maybe break this into 2 lines, one in OAP section (ui template api) and replace the UI in UI section ?

Reply:
I just resolved the conflict, which should have addressed this.